react/src/lib/Input/index.js

```diff
@@ -260,6 +264,10 @@ class Input extends React.Component {
 }
 
 Input.propTypes = {
+  /** @prop Text to display for blindness accessibility features | '' */
```

Comment:
If your default prop is null then it should be the word null on the other side of the pipe.
```
@prop Text to display for blindness accessibility features | null
```